### PR TITLE
Wps tools 1.0.3

### LINKED ISSUE #139: Address warnings
There are a number of warnings for Thunderbird, mostly related to deprecation. Related code should be replaced with newer code.

### LINKED ISSUE #140: Wps-tools 1.0.2 update
## Description
Update thunderbird with wps-tools 1.0.2

## Requested Action
reorganize module imports and add output_handling tools to demos

## Alternatives
None

### PR DESCRIPTION
**Pull request by cairosanders**
Resolves #140  and resolves #139 
- Reorganizes imports for `wps_tools` >1.0.0
- Adds `wps_tools.output_handling` to demos
- Updates `netCDF4` to remove most warnings

Test on port `30874`

### CODE REVIEW

File: thunderbird/processes/wps_generate_climos.py [hunk 1]
```diff
@@ -52,7 +47,7 @@ def __init__(self):
                 "Daily NetCDF Dataset",
                 abstract="NetCDF file",
                 min_occurs=1,
-                max_occurs=MAX_OCCURS,
+                max_occurs=1000,
```
Comment (nikola-rados) [inline thunderbird/processes/wps_generate_climos.py:50]: Did we get rid of the `MAX_OCCURS` variable?

File: thunderbird/utils.py [hunk 2]
```diff
@@ -51,3 +51,60 @@ def get_url():
...
+    get_robjects,
+    nc_to_dataset,
+    json_to_dict,
+    txt_to_string,
+)
+import requests
+from bs4 import BeautifulSoup
+
+
+def auto_construct_outputs(outputs):
```
Comment (nikola-rados) [inline thunderbird/utils.py:68]: Doesn't this method already exist in `wps_tools`? Why is it duplicated here?

**Review by nikola-rados — COMMENTED** [review]
Looks good, but I have a question below before approval.

File: thunderbird/utils.py (hunk 2 above)
Comment (cairosanders) [reply thunderbird/utils.py:68]: I will delete it I made it before I put it in wps_tools and forgot to delete

**Pull request merged**